using-mobile-extensions/mobile-core/mobile-core-api-reference.md

```diff
@@ -547,6 +547,32 @@ ACPCore.log(ACPMobileLogLevel.debug, tag: "MyClassName", message: "Provided data
 {% endtab %}
 {% endtabs %}
 
+### Handle open url action
+
+Mobile SDK allow the user to add a callback function which is triggered before [`open url action`](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#consequence-types) occurrs. If the callback function return YES, sdk will not operate `open url actoin`, otherwise continue `open url action`.
```

Comment:
Please reword so that it reads as follows:

"The Mobile SDKs allow users to add a callback function that is triggered before the [`open url action`](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#consequence-types) occurs. If the callback function returns YES, the SDKs do not complete the `open url action` action. If the callback function returns NO, the `open url action` is completed.

(Is my understanding correct for the last 2 sentences @yangyansong-adbe?)

Reply:
fixed.